packages/block-library/src/latest-posts/index.php

```diff
@@ -13,7 +13,7 @@
  * @return string Returns the post content with latest posts added.
  */
 function render_block_core_latest_posts( $attributes ) {
-	$recent_posts = wp_get_recent_posts(
+	$recent_posts_query = new WP_Query(
 		array(
 			'numberposts' => $attributes['postsToShow'],
```

Comment:
`numberposts` only works when using `get_posts()`, not `WP_Query`. For consistency I'd use `posts_per_page` though.

Reply:
changed to posts_per_page as using get_posts() now